gwcs/wcs.py

```diff
@@ -1513,25 +1512,113 @@ def to_fits_sip(self, bounding_box=None, max_pix_error=0.25, degree=None,
         that arise with high powers.
 
         """
-        if not isinstance(self.output_frame, cf.CelestialFrame):
+        # Other (Hidden) Parameters
+        # -------------------------
+        # _keep_axis_position : bool, optional
+        #     This parameter controls whether to keep all output axes in this
+        #     WCS object when creating FITS WCS or to reduce the number of
+        #     output axes to two celestial axes (lon/lat order is still
+        #     preserved). Default is False.
+        #
+        keep_naxis = kwargs.get('_keep_axis_position', False)
+        sep_frames_axes, axes_mapping = self._separable_frames_and_axes_groups()
+
+        # find CelestialFrame and make sure it has only one group of
+        # two separable axes:
+        for frame, axes_group in sep_frames_axes:
```

Comment:
OK, where is axes_group actually used?

Reply:
It is not used here (but it will be in different function, in a different PR which will need to decide how to split things for TAB). Here `axes_group` will be replaced with `_`.